Dockerfile

```diff
@@ -36,4 +36,4 @@ RUN useradd invenio --uid 1000 --gid 0 && \
     chown -R invenio:root ${WORKING_DIR}
 USER 1000
 
-CMD ["/usr/local/bin/gunicorn", "-b", ":8080", "--timeout", "120", "--graceful-timeout", "60", "invenio_app.wsgi_ui:application"]
+CMD ["/usr/local/bin/gunicorn", "-b", ":8080", "--access-logfile", "-", "--error-logfile", "-", "--timeout", "120", "--graceful-timeout", "60", "invenio_app.wsgi_ui:application"]
```

Comment:
Would it be useful to save the log to file rather than outputting to STDOUT? Or is this so we can see the output in OpenShift?

Reply:
this is so we can see the output in openshift. For now I don't think we need any logfile of webserver. Migration procedure has it's own logfile anyway.